lib/dug/runner.rb

```diff
@@ -33,6 +33,7 @@ def unprocessed_notifications(use_cache: true)
         @unprocessed_notifications = nil
       end
       unprocessed_label = servicer.labels(use_cache: use_cache)[Dug.configuration.unprocessed_label_name]
+      raise "Create the label '#{Dug.configuration.unprocessed_label_name}' on gmail" unless unprocessed_label
```

Comment:
Forgot to say... The configure step would check your current .yml and ensure the labels you have in there actually exist in Gmail and create them for you if they don't (maybe prompt by prompt)